robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowConfiguration.java

```diff
@@ -14,153 +14,5 @@
-      if (n != 0) return n;
-    }
-    n = realConfiguration.touchscreen - that.touchscreen;
-    if (n != 0) return n;
-    n = realConfiguration.keyboard - that.keyboard;
-    if (n != 0) return n;
-    n = realConfiguration.keyboardHidden - that.keyboardHidden;
-    if (n != 0) return n;
-    n = realConfiguration.hardKeyboardHidden - that.hardKeyboardHidden;
-    if (n != 0) return n;
-    n = realConfiguration.navigation - that.navigation;
-    if (n != 0) return n;
-    n = realConfiguration.navigationHidden - that.navigationHidden;
-    if (n != 0) return n;
-    n = realConfiguration.orientation - that.orientation;
-    if (n != 0) return n;
-    n = realConfiguration.screenLayout - that.screenLayout;
-    if (n != 0) return n;
-    n = realConfiguration.uiMode - that.uiMode;
-    if (n != 0) return n;
-    n = realConfiguration.screenWidthDp - that.screenWidthDp;
-    if (n != 0) return n;
-    n = realConfiguration.screenHeightDp - that.screenHeightDp;
-    if (n != 0) return n;
-    n = realConfiguration.smallestScreenWidthDp - that.smallestScreenWidthDp;
-    //if (n != 0) return n;
-    return n;
-  }
-
-  @Implementation
-  public boolean equals(Configuration that) {
-    if (that == null) return false;
-    if (that == realConfiguration) return true;
-    return realConfiguration.compareTo(that) == 0;
-  }
-
-  @Implementation
-  public boolean equals(Object that) {
-    try {
-      return equals((Configuration)that);
-    } catch (ClassCastException e) {
-    }
-    return false;
-  }
-
-  @Implementation
-  public int hashCode() {
-    int result = 17;
-    result = 31 * result + Float.floatToIntBits(realConfiguration.fontScale);
-    result = 31 * result + realConfiguration.mcc;
-    result = 31 * result + realConfiguration.mnc;
-    result = 31 * result + (realConfiguration.locale != null ? realConfiguration.locale.hashCode() : 0);
-    result = 31 * result + touchscreen;
-    result = 31 * result + realConfiguration.keyboard;
-    result = 31 * result + realConfiguration.keyboardHidden;
-    result = 31 * result + realConfiguration.hardKeyboardHidden;
-    result = 31 * result + realConfiguration.navigation;
-    result = 31 * result + realConfiguration.navigationHidden;
-    result = 31 * result + orientation;
-    result = 31 * result + screenLayout;
-    result = 31 * result + realConfiguration.uiMode;
-    result = 31 * result + realConfiguration.screenWidthDp;
-    result = 31 * result + realConfiguration.screenHeightDp;
-    result = 31 * result + realConfiguration.smallestScreenWidthDp;
-    return result;
-  }
-
-  @Implementation
-  public void setToDefaults() {
-    directlyOn(realConfiguration, Configuration.class).setToDefaults();
-    realConfiguration.screenLayout = Configuration.SCREENLAYOUT_LONG_NO |
-        Configuration.SCREENLAYOUT_SIZE_NORMAL;
-  }
-
-  @Implementation
-  public void setLocale( Locale l ) {
-    realConfiguration.locale = l;
-  }
-
-  public void overrideQualifiers(String qualifiers) {
```

Comment:
Yeah, we could to avoid breakages but calls to this method don't actually seem to do anything anyway. See previous comments about usage etc.